modules/custom/az_finder/src/Service/AZFinderVocabulary.php

```diff
@@ -94,6 +94,7 @@ public function addTermsTable(&$form_section, $vocabulary_id, $view_id, $display
           '' => $this->t('Default'),
           'expand' => $this->t('Expanded'),
           'collapse' => $this->t('Collapsed'),
+          'remove' => $this->t('Remove'),
```

Comment:
Consider "Hidden" to be less permanent

```suggestion
          'remove' => $this->t('Hidden'),
```